packages/loot-core/src/server/aql/compiler.ts

```diff
@@ -600,6 +626,14 @@ const compileFunction = saveStack('function', (state, func) => {
     }
 
     // date functions
+    case '$day': {
+      validateArgLength(args, 1);
+      return castInput(state, args[0], 'date');
+    }
+    case '$week': {
```

Comment:
I'm having trouble finding where this is used. I see us using `$day` in makeQuery, but not `$week`. Am I missing something?

Reply:
Good call out. I had it in there at first but removed the reference when I added the firstDayOfWeekIdx. Cheers!